components/TextFields/src/MDCTextInput.h

```diff
@@ -130,6 +130,17 @@
  */
 @property(nonatomic, nonnull, strong, readonly) UILabel *trailingUnderlineLabel;
 
+/**
+ An overlay view on the side of the input opposite from where reading and writing lines begin. In
+ LTR this means it will show on the Right side. In LTR, the Left side.
+ */
```

Comment:
I'm not sure re-explaining what "trailing" means is useful here. I think you can omit all but the first sentence.